### ISSUE TITLE
Compilation error with Elixir v1.3.0-rc.0

### ISSUE THREAD

**Issue by my-flow** [myfreeweb/httpotion#87]
```
==> httpotion
Compiling 1 file (.ex)

== Compilation error on file lib/httpotion.ex ==
** (ArgumentError) Access is not a protocol
    (elixir) lib/protocol.ex:76: Protocol.assert_protocol!/2
    lib/httpotion.ex:292: (module)
    (stdlib) erl_eval.erl:670: :erl_eval.do_apply/6
```

**Comment by myfreeweb** [comment]
Yeah the last release doesn't support 1.3. I really should release a new version now…

**Comment by myfreeweb** [comment]
Released 3.0.0.
